dashboard/lib/services/script_seed.rb

```diff
@@ -237,6 +237,10 @@ def self.destroy_outdated_objects(model_class, all_objects, imported_objects, se
     should_keep[true]
   end
 
+  def self.get_columns(model_class)
+    model_class.columns.map(&:name).map(&:to_sym) - %i(id)
```

Comment:
I'm going to leave us on the old activerecord-import version for now, since upgrading seems to have caused a performance regression for level seeding: https://github.com/code-dot-org/code-dot-org/pull/36912

Added a workaround here. This is based on the same workaround we're already using in `LevelLoader`.